R/drive_find.R

```diff
@@ -26,7 +26,7 @@
 #' By default, `drive_find()` does not include files in the trash: it adds `q =
 #' "trashed = false"` to the query. However, it will not do so if the user
 #' specifies a `q` search clause for trash inclusion or exclusion. To see only
-#' files in the trash, use [drive_view_trash()], which is a shortcut for
+#' files in the trash, use [drive_show_trash()], which is a shortcut for
```

Comment:
ah that's a good point - would it be preferable to have `drive_show_trash()` add a logical `trash` column (but not filter on the rows), and then leave `drive_view_trash()` to filter for only those in the trash?